cores/esp8266/libb64/cencode.cpp

```diff
@@ -23,10 +22,15 @@ void base64_init_encodestate_nonewlines(base64_encodestate* state_in){
   state_in->stepsnewline = -1;
 }
 
-char base64_encode_value(char value_in){
-  static const char encoding[] PROGMEM = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
-  if (value_in > 63) return '=';
-  return pgm_read_byte( &encoding[(int)value_in] );
+static inline char base64_encode_value(const char n) {
```

Comment:
This function was public, and now it's private to the TU. That is a breaking change. Any application or library that currently makes use of it won't compile anymore.

Reply:
well, is it really so that the libb64/* part is part of the public API? felt like a internal implementation accidentally leaking.

anyway, I reverted this part. although I would argue that libb64 is an implementation detail worth hiding..